DOCS-3028 Polish remaining TC2.1 Docs After Merge (#1247)
truenas/documentation: content/TrueCommand/Administration/Users.md

![Users Edit](/images/TrueCommand/2.0/UsersEditUser21.png "Users Edit")

There are several different user elements that you can configure, including their avatar, personal details, contact email address, team membership, and system permissions.

### User Details

Users or admins can add personal details about the user on this page. 

To go back to the original contents of the fields, click **RESET FORM** before you click **SAVE CHANGES**.

{{< tabs "User Options Tabs" >}}
{{< tab "Administrator" >}}
You can select the **TrueCommand Administrator** checkbox to designate the account as an administrator. 
{{< /tab >}}
{{< tab "Change Password" >}}

> You can change the account password by typing the new password into both the **Password** and **Password Confirm** fields. When prompted, enter the user's existing password. 

Reviewer: ```suggestion to remove the possessive user's with just user
You can change the account password by typing the new password into both the **Password** and **Password Confirm** fields. When prompted, enter the user existing password. 
```

Click **SAVE CHANGES** to make the change.
{{< /tab >}}
{{< tab "Email" >}}

> You can set up or change a user's email on this screen. If [SMTP]({{< relref "TrueCommand/Alerts/AlertManage.md" >}}) is not set up, an error message displays at the bottom of the screen stating **Failed to send email. Are your SMTP settings configured?**. Admins can click the **CONFIGURE** button to open the SMTP settings window. Before adding a user's email, go to **Alert Services** and verify you have set up the SMTP service.

Reviewer: ```suggestion to remove possessive user's with the following:
You can set up or change user email on this screen. If [SMTP]({{< relref "TrueCommand/Alerts/AlertManage.md" >}}) is not set up, an error message displays at the bottom of the screen stating **Failed to send email. Are your SMTP settings configured?**. Admins can click the **CONFIGURE** button to open the SMTP settings window. Before adding a user email, go to **Alert Services** and verify you have set up the SMTP service.
```

{{< /tab >}}
{{< tab "Two Factor Authentication" >}}

> You can set a user's [Two Factor Authentication]({{< relref "truecommand/tcgettingstarted/useraccounts.md" >}}), which requires them to enter a validation code emailed to them after they enter their username, password, and click **SIGN IN** on the login screen.

Reviewer: ```suggestion to remove possessive user's with non-possessive:
You can set user [Two Factor Authentication]({{< relref "truecommand/tcgettingstarted/useraccounts.md" >}}), which requires they enter a validation code emailed to them after they enter their username, password, and click **SIGN IN** on the login screen.
```

Click **JOIN TEAM** to display the list of existing teams, then select a team to add the user to it.
You can add users to multiple teams.
TrueCommand applies team permissions to any user added to a team, but setting specific permissions for users can override related team permissions.
Use the **Teams** screen to create new teams or edit existing ones.

### System Access

To limit non-administrative account access to connected systems, configure the **System Access** and **System Groups** sections.
You must first configure [system connections]({{< relref "ConnectingTrueNAS.md" >}}) and/or [system groups]({{< relref "ConnectingTrueNAS.md" >}}) in TrueCommand. Add systems from either the **Dashboard** or **Systems** screens.

Click **ADD SYSTEM** and select a system from the drop-down to give the user access to that system.
To restrict them to viewing system details, select the **read** permission.
To remove their access to a particular system, click **-** minus on that system.

When TrueCommand has system groups, the **ADD GROUP** button displays.
Click **ADD GROUP** and select a group from the drop-down to give the user access to all the systems in that group.

> To choose a user's group permissions, select **read** or **read/write**.

Reviewer: ```suggestion to remove possessive user's and replace with user
To choose the user group permissions, select **read** or **read/write**.
```

![Teams Edit](/images/TrueCommand/2.0/TeamsEdit.png "Teams Edit")

You can change a team profile avatar, name, or grant team members permission to create new TrueCommand alert rules by selecting the **Enable alert creation** checkbox.

The **Members** section shows which accounts are in the team.

> To add users to the team, click **ADD USER** and select users in the drop-down list.

Reviewer: ```suggestion
To add users to the team, click **ADD USER** and select users on the drop-down list.
```
